app/assets/stylesheets/pages.scss

```diff
@@ -16,16 +16,12 @@ header {
     padding: 50px 0;
 
     .btn-primary {
-      color: $white;
-      background-color: $brandGreen;
-      border-color: transparent;
+      // color: $white;
```

Comment:
why is it important to preserve this commented out code?

Reply:
am waiting for the new styleguide PR to go in first before I fix the buttons